### ISSUE TITLE
No power output?

### ISSUE THREAD

**Issue by q20** [edwork/homeassistant-peloton-sensor#60]
Hey there!

I was truly excited to test this out with home assistant, imagining all the silly stuff I can get up to with lights, sonos, etc. : )

After installing the integration, logging in and hitting "Just Ride", I see all the sensors light up (with a few seconds delay)... all sensors _except_ Power Output:

<img width="333" alt="image" src="https://user-images.githubusercontent.com/17024596/213770986-10c71d58-63e0-4dd4-893c-741b2b9e6649.png">

Is there any additional configuration required to get this metric displayed?
Thanks!

**Comment by edwork** [comment]
Hello! Thanks for using this integration - I'm sure this can be a quick fix done either later today or tomorrow. 

I'm unsure (on mobile currently) if this is a regression from a recent change or a change in the schema returned from Peloton but I'll squash this bug soon. 

- Ed

**Comment by q20** [comment]
Brilliant! Thanks, Ed.   👍

**Comment by edwork** [comment]
It's going to be just a day or so longer - I've run out of time to test this evening. 

Now that I'm off of the bike the sensor has populated but during the ride the data was missing. This and a few other issues should be cleaned up shortly.

**Comment by q20** [comment]
Yup, I've seen the same behaviour: a test ride shows the following:

![image](https://user-images.githubusercontent.com/17024596/214011606-aafb6250-b8af-4466-bdc1-f4b011bbf9aa.png)

Am I correct in guessing that Wh will collect cumulative output for a given session and _not_ display current power output?

**Comment by milleruk** [comment]
Did this bug get fixed at all as would love to have the current output showing so can do some lights and fan automations based on zones
